test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -335,15 +335,15 @@ object TestAspect extends TimeoutVariants {
    * An aspect that retries a test until success, with a default limit, for use
    * with flaky tests.
    */
-  val flaky: TestAspectAtLeastR[ZTestEnv with Annotations] =
+  val flaky: TestAspectAtLeastR[ZTestEnv with Annotations with Live] =
     flaky(100)
 
   /**
    * An aspect that retries a test until success, with the specified limit, for
    * use with flaky tests.
    */
-  def flaky(n: Int): TestAspectAtLeastR[ZTestEnv with Annotations] =
-    retry(Schedule.recurs(n))
+  def flaky(n: Int): TestAspectAtLeastR[ZTestEnv with Annotations with Live] =
+    retry(Schedule.recurs(n.toLong))
```

Comment:
Maybe we should overload for `Int` and `Long` so users don't have to do this everywhere?

Reply:
Agreed!